util/file.go

```diff
@@ -100,3 +99,13 @@ func CopyFileContents(src, dst string) (err error) {
 	err = out.Sync()
 	return
 }
+
+func prepareConfigFileDir(file string) (string, string, error) {
```

Comment:
We can also have a version of the test `"should be written and read successfully"` for this case.